.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
```

Comment:
yea, I used to use editorconfig for my project, this will setting my text editor to set 2 spaces and newline.